.github/workflows/release_js_sdk.yml

```diff
@@ -35,9 +38,14 @@ jobs:
       env:
         NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
 
+    - name: Update version
+      run: npm version ${{ github.event.inputs.version }} --no-git-tag-version
+
+    - name: Commit and push
+      run: git commit -am "Release ${{ github.event.inputs.version }}" && git push
```

Comment:
The git commit command should handle cases where there are no changes to commit:
```yaml
- name: Commit and push
  run: |
    if git diff --quiet; then
      echo "No changes to commit"
    else
      git commit -am "Release ${{ github.event.inputs.version }}" && git push
    fi
```